### DISCUSSION TITLE
Deeplink NOT SUPPORTING

### DISCUSSION THREAD

**Discussion by theadp** [vknow360/CustomWebView#8] [Q&A] [answered]
I am using version 10,when i click Whatsapp share button in my website through app it is not opening anything
![blocks (3)](https://user-images.githubusercontent.com/68984419/107475063-8924f100-6b99-11eb-9760-411d935f5220.png)

**Comment by vknow360** [comment, marked as answer]
v10.1 beta fixes this issue.
